public/js/app/components/__tests__/CourseFileLinks.test.js

```diff
@@ -83,6 +88,7 @@ describe('Component <CourseFileLinks>', () => {
       memoStorageURI: 'https://test.com/',
       courseRound: {
         roundId: '7',
```

Comment:
roundId has to be removed to be sure tests working properly

Reply:
Removed